nvflare/app_common/ccwf/ccwf_job.py

```diff
@@ -287,7 +303,11 @@ def add_cyclic(
             final_result_ack_timeout=client_config.final_result_ack_timeout,
         )
         self.to_clients(client_controller, tasks=["cyclic_*"])
-        self.to_clients(client_config.executor, tasks=["train", "validate", "submit_model"])
+
+        if not self.executor:
+            # We add the executor only if it's not added yet.
```

Comment:
it might be good if we have a way of warning the users that if one executor is added, then the following will be ignored.

Reply:
It might. But at this moment, let's go without since this is a very rarely used.